scripts/update/Updater.php

```diff
@@ -825,6 +828,20 @@ public function update($initialVersion)
         }
 
         $this->skip('6.12.0', '6.13.0.2');
+
+        if ($this->isVersion('6.13.0.2')) {
+            $service = new EntityChecksumCacheService([
+                EntityChecksumCacheService::OPTION_PERSISTENCE => 'default_kv',
+            ]);
+            $this->getServiceManager()->register(EntityChecksumCacheService::SERVICE_ID, $service);
+
+            /** @var EventManager $eventManager */
+            $eventManager = $this->getServiceManager()->get(EventManager::SERVICE_ID);
+            $eventManager->attach(ResourceDeleted::class, [EntityChecksumCacheService::SERVICE_ID, 'entityDeleted']);
+            $this->getServiceManager()->register(EventManager::SERVICE_ID, $eventManager);
```

Comment:
As I commented on manifest file we need caching checksums functionality only on VM and we never update an VM, it's always a fresh install. Therefore I think we don't need these services/event listeners registered in Updater script.

Reply:
Right! I've removed the setup from this extension and will create a backport PR in `taoOffline`.